### PR TITLE
added footer for index/signup

### PR DESCRIPTION
**Pull request by Pybite**
resoved issues #1 & #2

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -56,6 +56,8 @@ <h1>Watch the Video</h1>
     </div>
     </main>
     <script src="script.js"></script>
-    
+    <footer>
+      copyright &copy; Full Stack Web Dev
```
Comment (nbkhope) [inline src/index.html:60]:
```suggestion
      Copyright &copy; Full Stack Web Dev
```

File: src/signup.css [hunk 2]
```diff
@@ -66,4 +66,9 @@ form button[type='submit'] {
 .breadcrumb li a:hover {
   color: #01447e;
   text-decoration: underline;
+}
+
+footer {
```
Comment (nbkhope) [inline src/signup.css:71]: A page doesn't necessarily have one footer. Multiple sections of a website could have a footer as well. So we should use a class to style the footer.

`
<footer class="site-footer">
```

```
.site-footer {
```

Comment (Pybite) [reply src/signup.css:71]: oh okay ! will update the code

File: src/index.html [hunk 3]
```diff
@@ -56,6 +56,8 @@ <h1>Watch the Video</h1>
     </div>
     </main>
     <script src="script.js"></script>
-    
+    <footer>
+      Copyright &copy; Full Stack Web Dev
```
Comment (nbkhope) [inline src/index.html:60]: Usually when we have symbols we use the HTML entities / code like what you saw here.

https://www.w3schools.com/html/html_entities.asp

Out of curiosity if you really wanted to show a less-than sign, you would have to write &lt;

See the pattern? Always ampersand, abbreviation (`lt`, less than) and semicolon.

File: src/style.css [hunk 4]
```diff
@@ -71,3 +71,8 @@
   background-color: #95ffcd;
   text-align:center;
 }
+
+footer {
```
Comment (nbkhope) [inline src/style.css:75]: I see you are redefining the same footer styles here.

How about this? Let's create a file `global.css` that can be used to define styles that are global to all pages. Basically, if you need to style a footer, just link the global stylesheet and use the class. That way, you only define the class .site-footer in one place.

File: src/signup.css (hunk 2 above)
Comment (nbkhope) [reply src/signup.css:71]: Yeah:

> The <footer> tag defines a footer for a document or section.
> You can have several <footer> elements in one document.

Ref: https://www.w3schools.com/TAgs/tag_footer.asp

In truth, we could have used just `<div>`, but we should try to use semantic tags with meaning for every section. Visually there's no difference by default.

**Review by nbkhope — APPROVED** [review]

**Pull request merged**